processing/src/main/java/org/apache/druid/query/expression/LookupExprMacro.java

```diff
@@ -104,6 +105,15 @@ public ExpressionType getOutputType(InputBindingInspector inspector)
       @Override
       public String stringify()
       {
+        if (replaceMissingValueWith != null) {
+          return StringUtils.format(
+              "%s(%s, %s, '%s')",
+              FN_NAME,
+              arg.stringify(),
+              lookupExpr.stringify(),
+              replaceMissingValueWith
```

Comment:
This doesn't handle escaping properly. `replaceMissingValueWith` might contain `'` itself, or newlines, etc.

The simplest way to fix this is to keep the expr around and `stringify()` it here, like we do with `lookupExpr` (which is also required to be a literal).

Reply:
fixed